.github/workflows/cypress-tests.yml

```diff
@@ -0,0 +1,25 @@
+name: Cypress Tests
+'on':
+  push:
+    paths-ignore:
```

Comment:
Making only `.md` or `.yml` change isn't circumventing this action like I expected. One of the commits that only has markdown and config still triggered the tests.

Reply:
Hm, the latest md-only change re-ran the deploy script, not this one. Maybe the issue is with the pull-request yml.